src/vl-form.js

```diff
@@ -0,0 +1,67 @@
+import {nativeVlElement, define} from '/node_modules/vl-ui-core/dist/vl-core.js';
+
+/**
+ * VlForm
+ * @class
+ * @classdesc Formulier element.
+ *
+ * @extends HTMLElement
+ *
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-form/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-form/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-form.html|Demo}
+ *
+ */
+export class VlForm extends nativeVlElement(HTMLFormElement) {
+  static get _observedAttributes() {
+    return ['target', 'action'];
+  }
+
+  static get _targetElementId() {
+    return 'hidden-form-target';
+  }
+
+  connectedCallback() {
+    this._process();
+  }
+
+  get _targetElement() {
+    return this.querySelector(`iframe[name="${VlForm._targetElementId}"]`);
+  }
+
+  _getTargetElementTemplate() {
+    return this._template(`<iframe name="${VlForm._targetElementId}" width="0" height="0" border="0" hidden></iframe>`);
```

Comment:
Jammer dat dergelijk workaround de enige oplossing is. Zijn we zeker dat we met iframes geen security meldingen, ... kunnen krijgen?

Indien enige mogelijkheid moeten we niet kijken of we bv `sandbox` attribuut kunnen zetten of andere restricties kunnen opleggen aan deze iframe?

Reply:
Ik sta open voor alternatieve oplossingen. In ieder geval is het centraal trekken voordelig aangezien dan niet iedereen een eigen oplossing moet bedenken, ontwikkelen en onderhouden.

Reply:
Ik weet niet of andere oplossingen zijn... zou ik moeten onderzoeken... maar ik denk dat @ZhaodiFengVlaanderen al verschillende opties bekeken heeft.

Maar ik ga akkoord dat dit centraal moet zitten als het op deze manier moet.

Reply:
Er zijn verschillende oplossingen mogelijk om dit standaard gedrag van een form te omzeilen. Hier werd nu voor de iFrame oplossing gekozen, maar dat is eerder toevallig (of niet) ook de oplossing die jullie intern gekozen hadden.

Als er nu of in de toekomst elegantere oplossingen zijn, kunnen we dit uiteraard aanpassen.